phantazm-api/src/main/java/com/github/phantazmnetwork/api/config/SoundConfigProcessor.java

```diff
@@ -0,0 +1,46 @@
+package com.github.phantazmnetwork.api.config;
+
+import com.github.steanky.ethylene.core.ConfigElement;
+import com.github.steanky.ethylene.core.ConfigPrimitive;
+import com.github.steanky.ethylene.core.collection.ConfigNode;
+import com.github.steanky.ethylene.core.collection.LinkedConfigNode;
+import com.github.steanky.ethylene.core.processor.ConfigProcessException;
+import com.github.steanky.ethylene.core.processor.ConfigProcessor;
+import net.kyori.adventure.key.Key;
+import net.kyori.adventure.sound.Sound;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Objects;
+
+public class SoundConfigProcessor implements ConfigProcessor<Sound> {
```

Comment:
(i've also already made a SoundConfigProcessor in mapdata branch, will move to phantazm-commons if it's needed here as well)